kornia/geometry/liegroup/se3.py

```diff
@@ -248,6 +248,24 @@ def inverse(self) -> 'Se3':
         r_inv = self.r.inverse()
         return Se3(r_inv, r_inv * (-1 * self.t))
 
+    @classmethod
+    def random(cls, batch_size: Optional[int] = None, device=None, dtype=None) -> 'Se3':
+        """Create a Se3 group representing a random transformation.
+
+        Args:
+            batch_size: the batch size of the underlying data.
+
+        Example:
+            >>> s = Se3.random()
+            >>> s = Se3.random(batch_size=3)
+        """
+        r = So3.random(batch_size, device, dtype)
+        if batch_size is not None:
```

Comment:
```suggestion
        if batch_size is None:
          shape = ()
        else:
          shape = (batch_size,)
       return cls(r, rand(shape + (3,), device=device, dtype=dtype))
```
similar to https://github.com/kornia/kornia/pull/1987/files#diff-42adf0d69e7dd1e765e13b264d372bd6aa41fd7e2e274edf2ce76ba05a6bb82fR44